### ISSUE TITLE
[Request]: Yim menu ASI loader

### ISSUE THREAD

**Issue by vanijenkovBG** [Yimura/YimMenu#3275]
### Problem

The tittle explains it 

### Solution

Make and ASI loader like NENYOO had or implement something like SOAB into the menu

### Reason

I feel like the menu is missing some things

### Additional context

https://gta.2take1.menu/dev/mdsoab/

**Comment by tiCkuhAr** [comment]
Well, there is this.
https://github.com/maybegreat48/YimASI

**Comment by Rxann** [comment]
> Well, there is this.
> 
> https://github.com/maybegreat48/YimASI

I don't think that is maintained anymore. I thought there were plans of an Asi loader being added to the menu but it was closed for an unapparent reason.

**Comment by vanijenkovBG** [comment]
> Well, there is this. https://github.com/maybegreat48/YimASI

does it still work like can i try to compile it with the menu

**Comment by Rxann** [comment]
> > Well, there is this. https://github.com/maybegreat48/YimASI
> 
> does it still work like can i try to compile it with the menu

I haven't ever used it before, but I would not be surprised if it doesn't work.

**Comment by tiCkuhAr** [comment]
It worked last time I tried it about a week ago

**Comment by vanijenkovBG** [comment]
> It worked last time I tried it about a week ago

can u send the dll so i can try it too or just idk how to compile it

**Comment by tiCkuhAr** [comment]
I don't think sending DLLs is allowed.

**Comment by vanijenkovBG** [comment]
> I don't think sending DLLs is allowed.

u can send it in discord or somewhere else

**Comment by tiCkuhAr** [comment]
You will have to find someone else to help you with that.

**Comment by vanijenkovBG** [comment]
> You will have to find someone else to help you with that.

oh ok

**Comment by vanijenkovBG** [comment]
> You will have to find someone else to help you with that.

wait i wanna ask something do i have to put it in the menu or just compile the YimASI thing

**Comment by tiCkuhAr** [comment]
You build it, then put the WTSAPI32.dll in your GTA folder.

**Comment by vanijenkovBG** [comment]
> You build it, then put the WTSAPI32.dll in your GTA folder.

i did that but idk what to do now

**Comment by xesdoog** [comment]
@vanijenkovBG you can find a [compiled version here](https://github.com/xesdoog/YimASI) with a readme that explains everything you need to know. 
Also, yes it works without any issues.

**Comment by vanijenkovBG** [comment]
> @vanijenkovBG you can find a [compiled version here](https://github.com/xesdoog/YimASI) with a readme that explains everything you need to know. Also, yes it works without any issues.

i alr found it and the owner of yim extras addon put it in his script's update.bat if people want it with scripthoov
